apple-pie/styles/style.css

```
@@ -18,7 +18,95 @@ body, h1, h2, h3, h4, h5, h6, p, ol, ul {
 /*  Set the font */
 body {
   font-family: "Helvetica", sans-serif;
+  
 }
 
 /* STYLES */
-/* Write your CSS below */
\ No newline at end of file
+/* Write your CSS below */
+
+
+  @import url('https://fonts.googleapis.com/css2?family=Geologica:wght@300&display=swap');
+
+
+#firstImg { 
+  background-image: url('/images/apple-pie.jpg');
```

```suggestion
  background-image: url(../images/apple-pie.jpg);
```
En este caso como estamos dentro de styles, tenemos que decirle que salga una carpeta y luego entre en la de images